src/test/resources/assignment-v2/test-case-0.json

```diff
@@ -0,0 +1,2008 @@
+{
```

Comment:
Do we want to check these in? Before we had a pattern of getting the latest and greatest from the [shared test data repository](https://github.com/Eppo-exp/sdk-test-data) with `make test-data`.

Reply:
hmm yeah I was unclear about this. yeah I think I can remove this!